src/Diagnostics/Test/Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -53,7 +53,7 @@ protected static DiagnosticResult GetGlobalResult(DiagnosticDescriptor rule, par
             {
                 Id = rule.Id,
                 Severity = rule.DefaultSeverity,
-                Message = rule.MessageFormat.ToString()
+                Message = string.Format(rule.MessageFormat.ToString(), messageArguments)
```

Comment:
This is the only change I'm not sure about, since it is potentially changing the output of other tests. However, I guess if none of the tests break we're probably ok...